backend/api/src/main/java/com/virtuslab/gitmachete/backend/api/IGitMacheteRepositorySnapshot.java

```diff
@@ -29,10 +29,10 @@ public interface IGitMacheteRepositorySnapshot {
   Option<IExecutionResult> executeMachetePreRebaseHookIfPresent(IGitRebaseParameters gitRebaseParameters)
       throws GitMacheteException;
 
-  interface IOngoingRepositoryOperationInfo {
-    OngoingRepositoryOperation getOperationType();
+  interface IOngoingRepositoryOperation {
```

Comment:
For the problem with Interning Checker, look for `lombok-generated equals` string across the project

Reply:
It actually does not complain here.

Reply:
Okay it does. 😆